Src/ST-LIB_LOW/Sensors/EncoderSensor/EncoderSensor.cpp

```diff
@@ -1,7 +1,7 @@
 #include "Sensors/EncoderSensor/EncoderSensor.hpp"
 
 EncoderSensor::EncoderSensor(Pin pin1, Pin pin2, double *position, double* direction, double *speed, double *acceleration)
-: position(position), direction(direction), speed(speed), acceleration(acceleration){
+: position(position), direction(direction), speed(speed), acceleration(acceleration), idx_buff(0){
```

Comment:
TIP: when you declare it in the class you can do: `uint32_t idx_buff{0};` and that will initialize it, without having to explicitly list it out in the constructor